### PR TITLE
updating ReadMe for AKS Extension supporting CI/CD pipelines

### PR DESCRIPTION
**Pull request by pulkitaggarwl**

### CODE REVIEW

**Comment by Tatsinnit** [comment]
💡 Linked with this work: https://github.com/Azure/vscode-aks-tools/pull/42

File: README.md [hunk 1]
```diff
@@ -68,6 +68,15 @@ Running the AKS Periscope requires you to have a storage account associated with
 
 For more information on Diagnostics settings, visit [Create diagnostic settings to send platform logs and metics to different destinations](https://docs.microsoft.com/azure/azure-monitor/platform/diagnostic-settings).
 
+### Configure CI/CD 
+
+Right click on your AKS cluster and click on **Configure CI/CD** to create a CI/CD pipeline targetting your cluster.
```
Comment (Tatsinnit) [inline README.md:73]: 💡 Just for other folks as thought share, not sure if we want to underline that these 2 commands are the hooks for the extension `Deploy to Azure`? aka https://marketplace.visualstudio.com/items?itemName=ms-vscode-deploy-azure.azure-deploy

**Comment by Tatsinnit** [comment]
💡 Please note: you might have missed another place to add this commands which is line 6 or line 7 ( the bullet points which define what all commands fucntiaonlity does via this extension) 

image

File: README.md (hunk 1 above)
Comment (pulkitaggarwl) [reply README.md]: Expanding CI/CD makes sense.  However we should not need to take a dependency on Deploy to Azure extension for "Browse Pipelines". Why is that needed ? cc: @kanika1894

Comment (Tatsinnit) [reply README.md]: * Indeed let me correctly rephrase that out of 2 command only one I.e. configurePipeline is the one which has dependency and seems like browsePipeline is just url forwarding. 

* rectified that in my comment above, thank you 🙏

Comment (pulkitaggarwl) [reply README.md]: awesome. Have updated the readme for configure.

**Review by itowlson — APPROVED** [review]

**Pull request merged**